parameter_files/COLIBRE.yml

```diff
@@ -321,6 +325,7 @@ SOProperties:
     KineticEnergyGas: true
     KineticEnergyStars: true
     MassFractionSatellites: true
+    MassFractionExternal: true
```

Comment:
What is this property relevant to the PR?

Reply:
It was just missing from the COLIBRE parameter file, so I added it in. It was being calculated previously for COLBIRE

Reply:
COLIBRE*